packages/flutter/lib/src/material/bottom_sheet.dart

```diff
@@ -567,6 +572,8 @@ class _BottomSheetSuspendedCurve extends ParametricCurve<double> {
 /// to have a bottom sheet that has a scrollable child such as a [ListView] or
 /// a [GridView] and have the bottom sheet be draggable, you should set this
 /// parameter to true.
+/// When this parameter is set to true the top of [BottomSheet] will be padded
```

Comment:
This can be part of the same paragraph. Otherwise we usually put a line break in between paragraphs.
```suggestion
/// parameter to true. When this parameter is set to true the top of [BottomSheet] will be padded
```